frontend/src/metabase/dashboard/actions/save.js

```diff
@@ -121,10 +122,15 @@ export const updateDashboardAndCards = createThunkAction(
         dispatch(saveCardsAndTabs(updatedCardsAndTabs));
       });
 
-      // Make two parallel requests: one to update the dashboard and another for the dashcards and tabs
+      // Make two parallel requests: one to update the dashboard (without dashcards and tabs)
+      // and another for the dashcards and tabs
+      const onlyDashboard = produce(dashboard, d => {
```

Comment:
I'd rename it to something like `dashboardWithoutTabsAndDashcards`

Reply:
Yeah, I agree. Thankfully I'm going to be removing this code in the next PR so I will clean it up then to avoid rerunning CI if I don't have to